scripts/reloader.bash

```diff
@@ -20,9 +30,10 @@ fi
 
 
 if [ ! -z "${2}" ] && [[ "${2}" =~ ^(aliases|completion|plugins)$ ]] && [ -d "${2}/enabled" ]; then
-  # TODO: We should warn users they're using legacy enabling
+  _log_warning "Using legacy enabling, please update your bash-it version and migrate"
   for _bash_it_config_file in $(sort <(compgen -G "./${2}/enabled/*.bash")); do
     if [ -e "$_bash_it_config_file" ]; then
+      _set-prefix-based-on-path "${_bash_it_config_file}"
       # shellcheck source=/dev/null
       source "$_bash_it_config_file"
```

Comment:
Same here, this could benefit from additional debug logging as well.

Reply:
done